beacon-chain/rpc/beacon/validators.go

```diff
@@ -804,8 +804,9 @@ func (bs *Server) GetValidatorPerformance(
 
 	headState, err := bs.HeadFetcher.HeadState(ctx)
 	if err != nil {
-		return nil, status.Error(codes.Internal, "Could not get head state")
+		return nil, status.Errorf(codes.Internal, "could not get head state: %v", err)
```

Comment:
```suggestion
		return nil, status.Errorf(codes.Internal, "Could not get head state: %v", err)
```
This is a server error returned to our rpc clients, so it should be capitalized.